miio/fan_miot.py

```diff
@@ -141,6 +153,58 @@ def child_lock(self) -> bool:
         return self.data["child_lock"]
 
 
+class FanStatusC1(DeviceStatus):
+    """Container for status reports for Xiaomi Mi Smart Pedestal Fan DMaker C1."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    @property
+    def power(self) -> str:
+        """Power state."""
+        return "on" if self.data["power"] else "off"
+
+    @property
+    def is_on(self) -> bool:
+        """True if device is currently on."""
+        return self.data["power"]
+
+    @property
+    def mode(self) -> OperationMode:
+        """Operation mode."""
+        return OperationMode[OperationModeMiot(self.data["mode"]).name]
+
+    @property
+    def speed(self) -> int:
+        """Speed of the motor."""
+        return self.data["fan_level"]
+
+    @property
+    def oscillate(self) -> bool:
+        """True if oscillation is enabled."""
+        return self.data["swing_mode"]
+
+    @property
+    def delay_off_countdown(self) -> int:
+        """Countdown until turning off in seconds."""
+        return self.data["power_off_time"]
+
+    @property
+    def led(self) -> bool:
+        """True if LED is turned on, if available."""
```

Comment:
Considering "if available", should this use `self.data.get("light")` and be typed as `Optional[bool]`?